### PR TITLE
Don't create so many MediaStreams

### PR DESCRIPTION
**Pull request by markandrus**
@manjeshbhargav Fix for https://bugzilla.mozilla.org/show_bug.cgi?id=1405893

### CODE REVIEW

**Comment by markandrus** [comment]
I canceled Safari tests on Travis, hence the Xs. Firefox is hanging on Appveyor for some reason.

File: lib/getstats.js [hunk 1]
```diff
@@ -64,6 +64,14 @@ function _getStats(peerConnection, options) {
  * @returns {Array<MediaStreamTrack>}
  */
 function getTracks(peerConnection, kind, localOrRemote) {
+  var getSendersOrReceivers = localOrRemote === 'local' ? 'getSenders' : 'getReceivers';
+  if (peerConnection[getSendersOrReceivers]) {
+    return peerConnection[getSendersOrReceivers]().map(function(senderOrReceiver) {
+      return senderOrReceiver.track;
+    }).filter(function(track) {
+      return track;
```
Comment (manjeshbhargav) [inline lib/getstats.js]: @markandrus Seems like you're expecting some `track`s to be `null` here. Am I right?

Comment (markandrus) [reply lib/getstats.js]: Yes, the RTCRtpSender's `track` attribute is nullable (see [here](http://w3c.github.io/webrtc-pc/#rtcrtpsender-interface)).

**Comment by manjeshbhargav** [comment]
@markandrus One question, otherwise LGTM.

**Pull request merged**